server/controllers/shared/sharedController.ts

```diff
@@ -113,7 +113,6 @@ export default class SharedController {
         reportId: req.params.reportId,
         ...this.templateValues,
         data: {
-          ...this.data,
```

Comment:
Found this issue where we're not interested in saved data on POST, we only need what the user has submitted in the formData.